uber/config.py

```diff
@@ -111,6 +111,34 @@ def get_attendee_price(self, dt):
     def get_group_price(self, dt):
         return self.get_attendee_price(dt) - self.GROUP_DISCOUNT
 
+    def get_donation_tier_price(self, opt, dt):
```

Comment:
This (theoretically...) adds support for each tier to change in price at a certain date, much the way that attendee badges do now. This was added for use in Anthrocon.